docs/flow/config-language/files.md

```diff
@@ -6,3 +6,18 @@ weight: 100
 ---
 
 # Files
+River files are plaintext files with the `.river` file extension. Each River
+file may be referred to as a 'configuration file', or a 'River configuration'.
+
+River files are required to be UTF-8 encoded, and are permitted to contain
+Unicode characters. River files can use both Unix-style line endings (LF) and
+Windows-style line endings (CRLF), but formatters may replace all line endings
+with Unix-style ones.
+
+There is experimental support for River in
+[vim](https://github.com/rfratto/vim-river) and in
+[VSCode](https://github.com/rfratto/vscode-river).
```

Comment:
Can we put tooling like this in its own header and call out that it's community tooling?